src/main/java/foundation/privacybydesign/bigregister/RestApi.java

```diff
@@ -98,9 +98,24 @@ public Response getAttributes(String disclosureJWT) throws KeyManagementExceptio
             return Response.status(Response.Status.INTERNAL_SERVER_ERROR).entity("Could not parse date").build();
         }
 
+        String bigNumber;
+        try {
+            bigNumber = new BIGWebSearch().getBIGNumber(familyName, dateOfBirth, gender);
+        } catch (BIGRequestException e) {
+            // This should indicate a problem on their end or with the connection, not on our side.
+            System.out.println("BIG request error: " + e.getMessage());
```

Comment:
Nitpick: this should be `logger.error`, but as previously discussed this is probably originally my mistake.